Lil-Peep/src/styles/modules/description-banner.scss

```diff
@@ -0,0 +1,32 @@
+.banner-section {
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.banner-content {
+  background: linear-gradient(90deg,
+    rgba(200, 190, 183, 1),
+    rgba(200, 190, 183, 1) 30%,
+    rgba(200, 190, 183, 1) 30%,
+    rgba(200, 190, 183, 0));
+  color: $color-white;
+  display: flex;
+  flex-direction: column;
+  height: 250px;
+  justify-content: center;
+  text-align: center;
+  width: 50%;
+
+  p {
+    font-family: $Lato;
```

Comment:
Name of variable `Lato` should be written in all lowercase letters with hyphens instead of underscores